modules/post/linux/manage/busybox_jailbreak.rb

```diff
@@ -0,0 +1,85 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class Metasploit3 < Msf::Post
+
+  def initialize
+    super(
+      'Name'         => 'BusyBox Jailbreak ',
+      'Description'  => 'This module will send a set of commands to a open
+                         session that is connected to a BusyBox limited shell
+                         (i.e. a router limited shell). It will try different
+                         known tricks to try to jailbreak the limited shell and
+                         get a full sh busybox shell.',
+      'Author'       => 'Javier Vicente Vallejo',
+      'License'      => MSF_LICENSE,
+      'References'   =>
+        [
+          [ 'URL', 'http://vallejo.cc']
+        ],
+      'Platform'      => ['linux'],
+      'SessionTypes'  => ['shell']
+    )
+  end
+
+
+
+  def run
+
+    bfound = false
+
+    ############################################################method 1
```

Comment:
These comments aren't adding any useful information and should be deleted, please.